src/controllers/userController.js

```diff
@@ -0,0 +1,70 @@
+import userService from '../services/userService';
+import { jwtSignUser } from '../utils/index';
+import { comparePassword } from '../helpers/comparePassword';
+import { logins } from '../models';
+/**
+ * @class UsersController
+ */
+class UserController {
+  /** Login User
+   * @static
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   * @returns {string} loginUsers
+   */
+  static async loginAUser(req, res) {
+    const { email, password } = req.body;
+    try {
+      const loggedUser = await userService.loginAUser(email);
+      if (!loggedUser) {
+        return res.status(404).json({
+          error: 'Invalid login details',
+        });
+      }
+      const userDetail = {
+        id: loggedUser.dataValues.id,
+        email: loggedUser.dataValues.email,
+      };
+      if (loggedUser) {
+        const validPassword = await comparePassword(
+          loggedUser.password,
+          password
+        );
+        if (validPassword) {
+          const lastLogin = await userService.findLastLogin(
```

Comment:
You don't have to fetch the database for lastLogin to update it. Also lastlogin will always return a value, so this conditional will always be true.

Reply:
Ok, thanks